### ISSUE TITLE
Any iPhone client supporting XEP-0357?

### ISSUE THREAD

**Issue by jans23** [royneary/oshiya#4]
Hi!
Since I have no other way to reach you I'm submitting this issue. I'm very excited reading about your progress on implementing XEP-0357. Are you aware of any iPhone client supporting XEP-0357?

**Comment by weiss** [comment]
> Are you aware of any iPhone client supporting XEP-0357?

Yes, ChatSecure [supports it now](https://chatsecure.org/blog/chatsecure-v323-xmpp-push/).
